src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -26,6 +26,9 @@ public void isNonZeroUnsignedInteger() {
         // EP: zero
         assertFalse(StringUtil.isNonZeroUnsignedInteger("0"));
 
+        // EP: negative zero
+        assertFalse(StringUtil.isNonNegativeInteger("-0"));
```

Comment:
Maybe this should be isNonZeroUnsignedInteger instead since this is in its test case?